virtcontainers/device/drivers/vfio.go

```diff
@@ -194,6 +194,7 @@ func getVFIODetails(deviceFileName, iommuDevicesPath string) (deviceBDF, deviceS
 	case config.VFIODeviceNormalType:
 		// Get bdf of device eg. 0000:00:1c.0
 		deviceBDF = getBDF(deviceFileName)
+		deviceSysfsDev = filepath.Join(config.SysBusPciDevicesPath, deviceFileName)
```

Comment:
Looks like the sysfs path is only required by cloud hypervisor. A BDF is sufficient for other hypervisors. 
I would add a comment explaining this.